packages/atomic/src/components/facets-v1/atomic-numeric-facet/atomic-numeric-facet.tsx

```diff
@@ -138,12 +156,28 @@ export class AtomicNumericFacet
     );
   }
 
+  private areRangesEqual(
+    firstRange: NumericRangeRequest,
+    secondRange: NumericRangeRequest
+  ) {
+    return (
+      firstRange.start === secondRange.start &&
+      firstRange.end === secondRange.end &&
+      firstRange.endInclusive === secondRange.endInclusive
+    );
+  }
+
   private renderValue(facetValue: NumericFacetValue, onClick: () => void) {
-    // TODO: implement formats
-    const displayValue = this.bindings.i18n.t('to', {
-      start: facetValue.start,
-      end: facetValue.end,
-    });
+    const manualRangeLabel = this.manualRanges.find((range) =>
+      this.areRangesEqual(range, facetValue)
+    )?.label;
+    const displayValue = manualRangeLabel
```

Comment:
if there's a label, it will take priority, if there's a special format, it will take 2nd priority. If you have no label and format, the basic number format will be applied